.test-defs/provider-vsphere.yaml

```
@@ -12,4 +12,4 @@ spec:
       go run -mod=vendor ./test/tm/generator.go
       --infrastructure-provider-config-filepath=$INFRASTRUCTURE_PROVIDER_CONFIG_FILEPATH
       --controlplane-provider-config-filepath=$CONTROLPLANE_PROVIDER_CONFIG_FILEPATH
-  image: golang:1.17.11
+  image: golang:1.18.4
```

let's use the latest version 
```suggestion
  image: golang:1.18.5
```